src/main/kotlin/graphql/kickstart/tools/resolver/FieldResolverScanner.kt

```diff
@@ -14,6 +14,7 @@ import org.slf4j.LoggerFactory
 import java.lang.reflect.Method
 import java.lang.reflect.Modifier
 import java.lang.reflect.Type
+import java.util.*
```

Comment:
Don't do wildcard imports.

Reply:
done

Reply:
Feel free to mark my suggestions as resolved, doesn't seem like I can do this.